lib/inets/doc/src/httpd.xml

```diff
@@ -128,6 +123,13 @@
 	<p>Default is <c>any</c></p>
       </item>
 
+      <tag><marker id="prop_server_name"></marker>{server_name, string()}</tag>
+      <item>
+	<p>The name of your server, normally a fully qualified domain name.</p>
+        <p>If not given, this defaults to the fully qualified hostname as obtained
+          from <c>net_adm:localhost()</c>.</p>
+      </item>
```

Comment:
We should probably only say defaults to net_adm:local_host(), as it will only be a qualified hostname if you started the node with -name if started with -sname it will not be qualified.  But we could leave that for the net_adm:localhost doc :)

Reply:
Good point. I updated my commit to just mention it defaults to `net_adm:localhost()`.